ui/components/component-library/button-base/button-base.js

```diff
@@ -133,7 +133,7 @@ ButtonBase.propTypes = {
   /**
    * Additional props to pass to the Text component that wraps the button children
    */
-  buttonTextProps: PropTypes.shape(Text.propTypes),
+  buttonTextProps: PropTypes.object,
```

Comment:
PropTypes will be resolved in TS migration of this component https://github.com/MetaMask/metamask-extension/issues/18886